sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/system/Utils.groovy

```diff
@@ -0,0 +1,47 @@
+package life.qbic.portal.sampletracking.system
+
+import com.liferay.portal.model.User
+import com.vaadin.server.VaadinService
+import life.qbic.datamodel.dtos.portal.PortalUser
+import life.qbic.portal.utils.user.UserRelated
+
+/**
+ * <p>Utility class that provides access to some system context properties.</p>
+ *
+ * @since 1.0.0
+ */
+class Utils {
```

Comment:
```suggestion
class SystemContext {
```
I would suggest renaming this class to give it meaning. Utils is pretty generic and as far as I understood it we use this to handle the system context?